BoardingPass/BoardingNavigationController.swift

```diff
@@ -91,43 +161,43 @@ public class BoardingNavigationController: UINavigationController {
      - parameter animated: Specify true to animate the transition or false
      if you do not want the transition to be animated.
      */
-    public func popToPreviousViewController(animated animated: Bool) {
-        if let poppableViewController = topViewController.flatMap(boardingInfoAfter) {
-            popToAndInsertIfNeeded(poppableViewController, animated: animated)
+    public func popToPreviousViewController(animated: Bool) {
+        if let poppableViewController = topViewController.flatMap({ boardingInfo(afterController: $0) }) {
```

Comment:
#? I think `.flatMap( boardingInfo(afterController: ))` will work without having to wrap it in a closure